cmd/kyma/provision/gardener/aws/cmd.go

```diff
@@ -145,15 +136,15 @@ func newProvider(o *Options) (*types.Provider, error) {
 	p.CustomConfigurations["target_provider"] = "aws"
 	p.CustomConfigurations["disk_type"] = o.DiskType
 	p.CustomConfigurations["worker_minimum"] = o.ScalerMin
-	p.CustomConfigurations["worker_maximum"] = o.NodeCount
-	p.CustomConfigurations["worker_max_surge"] = o.Surge
-	p.CustomConfigurations["worker_max_unavailable"] = o.Unavailable
-	p.CustomConfigurations["vnetcidr"] = o.CIDR
-	p.CustomConfigurations["workercidr"] = o.WCIDR
-	p.CustomConfigurations["networking_nodes"] = o.NetworkNodes
-	p.CustomConfigurations["networking_pods"] = o.NetworkPods
-	p.CustomConfigurations["networking_services"] = o.NetworkServices
-	p.CustomConfigurations["networking_type"] = o.NetworkType
+	p.CustomConfigurations["worker_maximum"] = o.ScalerMax
+	p.CustomConfigurations["worker_max_surge"] = 1
+	p.CustomConfigurations["worker_max_unavailable"] = 1
+	p.CustomConfigurations["vnetcidr"] = "10.250.0.0/16"
+	p.CustomConfigurations["workercidr"] = "10.250.0.0/16"
+	p.CustomConfigurations["networking_nodes"] = ""
```

Comment:
Per the changes I made in Hydroform last time, you should not need to set these 3 networking properties. They default to empty string in the terraform template.

Reply:
Cool, I deleted them as well.